src/libs/auth.ts

```diff
@@ -137,7 +161,9 @@ export const hasBillingScope = () => {
     // For Google check the scope directly on the user object.
     [isGoogleAuthority(), () => _.includes('https://www.googleapis.com/auth/cloud-billing', getUser().scope)],
     // For B2C check the hasGcpBillingScopeThroughB2C field in the auth store.
-    () => authStore.get().hasGcpBillingScopeThroughB2C === true
+    // This is naughty - don't use `any`, but for now it's fine
+    // eslint-disable-line @typescript-eslint/no-explicit-any
+    () => (authStore.get() as any).hasGcpBillingScopeThroughB2C === true
```

Comment:
Now that authStore's state is typed as `any`, I believe this and other casts are no longer needed.